test/karma.config.cjs

```diff
@@ -21,7 +21,7 @@ module.exports = function(config) {
     port: 9876,
     colors: true,
     logLevel: config.LOG_INFO,
-    browsers: ['ChromeHeadless'],
+    // browsers: ['ChromeHeadless'],
```

Comment:
Was this change intentional?

Reply:
Was just for my own testing. I pushed with breaking tests. I'll put it back :)